### DISCUSSION TITLE
Package reference from a component

### DISCUSSION THREAD

**Discussion by lorenzovecchietti** [ansys/pyedb#386] [Q&A] [answered]
How can I get the package reference associated with a component?
![immagine](https://github.com/ansys/pyedb/assets/58366962/e7ba8c43-87d6-4874-a8a2-2add1456f84e)

**Comment by hui-zhou-a** [comment, marked as answer]
@lorenzovecchietti you can get the package def name by property pacakge def

`edbapp.components["J5"].package_def`
